cpp/source/deal.II/supercapacitor.templates.h

```diff
@@ -461,7 +461,7 @@ void SuperCapacitor<dim>::load(const std::string &filename)
   std::shared_ptr<boost::property_tree::ptree> geometry_database =
       std::make_shared<boost::property_tree::ptree>(
           _ptree.get_child("geometry"));
-  _geometry->repartition(geometry_database);
+  _geometry->repartition();
```

Comment:
it uses `_weights` which is read from the ptree so this would not change the current behavior.